examples/sgemm/SGEMM_kernels.ispc

```diff
@@ -403,7 +412,7 @@ static task void SGEMM_tileNoSIMDIntrin_task(uniform float matrixA[], uniform fl
 export void SGEMM_tileNoSIMDIntrin_withTasks(uniform float matA[], uniform float matB[], uniform float matC[],
     uniform int M, uniform int N, uniform int K) {
     // Generate more tasks than threads, to enable some load balancing:
-    uniform int numTasks = num_cores()*4;
+    uniform int numTasks = M / programCount;
```

Comment:
Some use ``K / programCount``, some use ``M / programCount``. I think it requires a note in comments to explain this.

Reply:
True. Fixed.